zulipterminal/config/keys.py

```diff
@@ -332,22 +332,22 @@ class KeyBinding(TypedDict):
     },
     'BEGINNING_OF_LINE': {
         'keys': ['ctrl a', 'home'],
-        'help_text': 'Jump to the beginning of line',
+        'help_text': 'Jump to the beginning of the current line',
         'key_category': 'editor_navigation',
     },
     'END_OF_LINE': {
         'keys': ['ctrl e', 'end'],
-        'help_text': 'Jump to the end of line',
+        'help_text': 'Jump to the end of the current line',
         'key_category': 'editor_navigation',
     },
     'ONE_WORD_BACKWARD': {
         'keys': ['meta b', 'shift left'],
-        'help_text': 'Jump backward one word',
+        'help_text': 'Jump backward to the start of current or previous word',
```

Comment:
I'm not entirely convinced of all of these - they're clearer wording in some cases, but also longer. More compact and concise would make these clearer.

For the navigation text, perhaps we can simplify using the fact that they all start with 'Jump'? That common phrasing suggests a simplification.

The other reason to keep these shorter is that they make the help menu wider or start to wrap, though currently there is one command that is even longer than these.